pkg/timeutils/json_test.go

```diff
@@ -0,0 +1,46 @@
+package timeutils
+
+import (
+	"testing"
+	"time"
+)
+
+func TestFastMarshalJSONWithInvalidDate(t *testing.T) {
+	aTime := time.Date(-1, 1, 1, 0, 0, 0, 0, time.Local)
```

Comment:
Very minor thing, but could you add a comment that its the 'year' field that's invalid (-1, & 10000)? just so that people who review this can know what's going on w/o having to check the golang docs to know which of the 8 fields you're passing in are invalid.
```
// Testing to ensure `year` fields is between 0 and 9999
```